tests/antithesis/README.md

```diff
@@ -7,7 +7,7 @@ This directory enables integration of Antithesis with etcd. There are 4 containe
 Run this command from the `antithesis/test-template` directory:
 
 ```bash
-docker build . -f Dockerfile.client -t etcd-client:latest
+docker build . -f Dockerfile -t etcd-client:latest
```

Comment:
This is missing change in `.github/workflows/antithesis-test.yml` but let's merge it as it's not breaking anyone.